acceptance_tests/features/social_sign_in_internal.feature

```diff
@@ -0,0 +1,40 @@
+@social
+Feature: Internal user signs in to social UI
+  As an internal user
+  I need to sign in to the social SDC system
+  So that I can access my SDC account and carry out a number of actions
+
+  Scenario: User signs in correctly (social)
```

Comment:
Is `(social)` added to the end of each step to allow us to identify failures more easily in logs?

Reply:
Mainly it's there because the features are identical to the r-ops login/logout features, but the strings need to be different otherwise it will call the same steps.

Reply:
Could they be better written, e.g.: "User signs into social response operations"?

Reply:
I did for one or two of them but most of the scenario steps don't reference which UI they are about so this seemed the cleanest way to word them distinctly.